examples/HastyNoise/config.cmake

```diff
@@ -0,0 +1,8 @@
+hunter_config(Boost VERSION ${HUNTER_Boost_VERSION}
+    CMAKE_ARGS
+        BUILD_SHARED_LIBS=ON 
+        Boost_USE_STATIC_LIBS=OFF
+        Boost_USE_MULTITHREADED=ON
+        Boost_USE_STATIC_RUNTIME=OFF
```

Comment:
Same, will be set automatically:
- https://github.com/ruslo/hunter/blob/v0.23.203/cmake/templates/BoostConfig.cmake.in#L42-L49